kernel/arch/dreamcast/hardware/pvr/pvr_buffers.c

```diff
@@ -191,7 +191,16 @@ void pvr_allocate_buffers(pvr_init_params_t *params) {
     /* Look at active lists and figure out how much to allocate
        for each poly type */
     opb_total_size = 0;
+
+    /**
```

Comment:
I don't think this is meant to be a doxygen comment...

```suggestion
    /*
```